bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/utils/StringUtils.java

```diff
@@ -148,6 +148,14 @@ public static BArray fromStringArray(String[] s) {
         return new ArrayValueImpl(bStringArray, false);
     }
 
+    public static BString[] getBStringArray(String[] s) {
```

Comment:
Do we need public api for this? Can we use internal package?

Reply:
Moved this method to an internal utils class for the test package.